buildbot/rotation.csv

```diff
@@ -10,359 +10,359 @@ Date,User
 2019-12-11,dibyo
 2019-12-12,christiewilson
 2019-12-13,sbws
-2019-12-14,jasonhall
+2019-12-14,
 2019-12-15,
-2019-12-16,
+2019-12-16,jasonhall
 2019-12-17,vdemeest
 2019-12-18,andrea.frittoli
 2019-12-19,dibyo
 2019-12-20,christiewilson
-2019-12-21,sbws
-2019-12-22,jasonhall
-2019-12-23,
-2019-12-24,
-2019-12-25,vdemeest
-2019-12-26,andrea.frittoli
-2019-12-27,dibyo
-2019-12-28,christiewilson
-2019-12-29,sbws
+2019-12-21,
+2019-12-22,
+2019-12-23,vdemeest
+2019-12-24,andrea.frittoli
```

Comment:
Uhm, not likely :P

Reply:
lol :hugs: 
We can remove those or just not caring about them :angel: